### PR TITLE
[IMP] fertinova: Add subtotal fields for quantities

### PR DESCRIPTION
**Pull request by edy1192**

### CODE REVIEW

File: fertinova/views/sale_views.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_order_form_inherit" model="ir.ui.view">
+        <field name="name">sale.order.form.inherit</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='order_line']/tree//field[@name='price_subtotal']" position="after">
+                <field name="subtotal_qty_delivered" widget="monetary" sum="SubTotal Delivered"/>
```
Comment (luisg123v) [inline fertinova/views/sale_views.xml]: Wrong indentation

File: fertinova/models/sale.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    @api.depends('qty_delivered', 'qty_to_invoice', 'qty_invoiced')
+    def _compute_subtotal_qtys(self):
+        for line in self:
+            line.subtotal_qty_to_invoice = line.price_unit * \
```
Comment (luisg123v) [inline fertinova/models/sale.py]: Prefer parenthesis over backslashes

File: fertinova/views/sale_views.xml (hunk 1 above)
Comment (edy1192) [reply fertinova/views/sale_views.xml]: Thanks! Done 😄

File: fertinova/models/sale.py (hunk 2 above)
Comment (edy1192) [reply fertinova/models/sale.py]: Thanks! Done 😄

File: fertinova/views/sale_views.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_order_form_inherit" model="ir.ui.view">
+        <field name="name">sale.order.form.inherit</field>
+        <field name="model">sale.order</field>
+        <field name="inherit_id" ref="sale.view_order_form"/>
+        <field name="arch" type="xml">
```
Comment (luisg123v) [inline fertinova/views/sale_views.xml]: Wrong indentation, you've got eight spaces more than the previous node.

Comment (edy1192) [reply fertinova/views/sale_views.xml]: Thanks, done.

**Pull request merged**